lib/vagrant-service-manager/command.rb

```diff
@@ -253,6 +264,28 @@ def print_vagrant_box_version(script_readable = false)
           end
         end
       end
+
+      def restart_service(service)
+
+        if SCCLI_SERVICES.include? service
+          # since sccli understand k8s
+          service = "k8s" if service == "kubernetes"
```

Comment:
@LalatenduMohanty : This line will be removed after we have support for `k8s` and `kubernetes` both in `sccli`. Here we are accepting both the terms for kubernetes service.